packages/react-cosmos-config/src/index.js

```diff
@@ -1,11 +1,12 @@
 import path from 'path';
+import slash from 'slash';
 import resolveFrom from 'resolve-from';
 import importModule from 'react-cosmos-utils/lib/import-module';
 
 const resolveUserPath = (userPath, rootPath) =>
-  path.isAbsolute(userPath) ? userPath : (
+  slash(path.isAbsolute(userPath) ? userPath : (
```

Comment:
> path.join already uses the platform specific separator

@ovidiubute yes, but it doesn't normalize the existing slashes from the concatenated paths. Am I right?